src/style-stars--4.css

```diff
@@ -0,0 +1,3 @@
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(../stars/yellow_star.png);
+}
```

Comment:
According to the checklist, avoid using tag names for styling, except for `html` and `body`. Instead, style elements using `.classes`. If the `.stars__star` is a class applied to a `span` or `div`, consider using only the class for styling.